build-test.sh

```diff
@@ -257,10 +257,6 @@ build_Tests()
         build_MSBuild_projects "Restore_Product" "${__ProjectDir}/tests/build.proj" "Restore product binaries (build tests)" "-BatchRestorePackages"
     fi
 
-    if [ -n "$__BuildAgainstPackagesArg" ]; then
-        build_MSBuild_projects "Tests_GenerateRuntimeLayout" "${__ProjectDir}/tests/runtest.proj" "Restore product binaries (run tests)" "-BinPlaceRef" "-BinPlaceProduct" "-CopyCrossgenToProduct"
-    fi
-
```

Comment:
Since we're removing this, could you get rid of the buildagainstpackages argument?

Reply:
Yes thank you I missed this.

Reply:
Addressed